From 6bd9add616720f01d9d061f35dcdf529ad903d6c Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 20 Jul 2017 05:15:58 +0200
Subject: [PATCH 1/1] efi_console: use EFIAPI for callback functions

The call to efi_create_event requires notification functions
flagged as EFIAPI.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_console.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lib/efi_loader/efi_console.c b/lib/efi_loader/efi_console.c
index f37d457847..13279d3f48 100644
--- a/lib/efi_loader/efi_console.c
+++ b/lib/efi_loader/efi_console.c
@@ -437,11 +437,12 @@ struct efi_simple_input_interface efi_con_in = {
 
 static struct efi_event *console_timer_event;
 
-static void efi_key_notify(struct efi_event *event, void *context)
+static void EFIAPI efi_key_notify(struct efi_event *event, void *context)
 {
 }
 
-static void efi_console_timer_notify(struct efi_event *event, void *context)
+static void EFIAPI efi_console_timer_notify(struct efi_event *event,
+					    void *context)
 {
 	EFI_ENTRY("%p, %p", event, context);
 	if (tstc())
-- 
2.13.2

